lms/services/hubspot/service.py

```diff
@@ -0,0 +1,90 @@
+from datetime import datetime
+
+from hubspot import HubSpot
+from sqlalchemy import select
+from sqlalchemy.exc import MultipleResultsFound
+
+from lms.models.hubspot import HubSpotCompany
+from lms.services.hubspot._client import HubSpotClient
+from lms.services.upsert import bulk_upsert
+
+
+class HubSpotService:
+    def __init__(self, db, client: HubSpotClient, region_code: str):
+        self._db = db
+        self._client = client
+        self._region_code = region_code
+
+    def get_company(self, organization_id: str) -> HubSpotCompany | None:
+        """Get the HubSpot company associated to the given LMS organization ID."""
+        try:
+            return self._db.scalars(
```

Comment:
What does `scalars` do in this context? I would expect that to be used in `SELECT COUNT...` and similar aggregate queries, where the result is a single primitive rather than a record to be mapped.

Reply:
Yeah the name is not great, scalar is not only scalar types (int, str) but the models themselves.